ci/run_compliance

```diff
@@ -0,0 +1,122 @@
+#! /usr/bin/env python3
```

Comment:
What's the primary reason for having a separate script?  Are there a lot of compliance tests?  I was hoping that we could fold these into our current flow without need of a separate script.

Reply:
Awk!   This is why we have reviews of pull-requests.    That file was not supposed to get pushed-in.  I hacked-it up quickly to write out the compliance regression in `.metrics.json format`.  Nobody was ever supposed to see it...